tap-gui/configurator/create-plug-in-wrapper.hbs.md

```diff
@@ -343,12 +339,16 @@ To update your dependencies for the specific Backstage plug-in you want to wrap:
    yarn install --ignore-engines
    ```
 
+### <a id="remove-code"></a> Remove unnecessary code for the back-end plug-in
```

Comment:
There's two <a id="remove-code"> in this file, may want to differentiate with -frntnd-plgn and -bcknd-plgn